### DISCUSSION TITLE
Table value is an observable array

### DISCUSSION THREAD

**Discussion by pk1201** [primefaces/community#772] [PrimeNG] [answered]
Hi I'm getting this below error when I tried to use the 

<p-table  [value]="(fundStreams$ | async)!" [columns]="cols" responsiveLayout="scroll" [rows]="10"></p-table>

Cannot assign to read only property '0' of object '[object Array]'

How do we address this, please help.

using 
angular 16.1.0
PrimeNG 16.9.1

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Create a logic to assign the value to the variable first, it'll work.
